lib/generators/scenic/view/templates/db/migrate/create_view.erb

```diff
@@ -1,5 +1,5 @@
 class <%= migration_class_name %> < ActiveRecord::Migration
   def change
-    create_view :<%= plural_name %><%= ", materialized: true" if materialized? %>
+    create_view "<%= plural_name %>"<%= ", materialized: true" if materialized? %>
```

Comment:
Since it shows up a few times, maybe add it to the view_generator as a helper that returns the quoted or symboled name?

`create_view <%= view_name %>` or whatever (I don't have a good name offhand).